.env

```diff
@@ -0,0 +1,7 @@
+apiKey: "AIzaSyCrcLMbeWfPvHwAfGMtNDM8NonUH5l4yGY",
```

Comment:
don't commit this file. create a .sample.env file and commit that for reference and testing. add this file to gitignore